### PR TITLE
[ICDS Dashboard]Change default behaviour of "Child Beneficiary List" to include children till 6 years 

### PR DESCRIPTION
**Pull request by czue**
https://dimagi-dev.atlassian.net/browse/ICDS-796

the first commit is a completely unrelated cleanup/change made to a totally different class that i thought was the relevant report.

the second commit is the actual change.

Note that I have not tested/run these locally.

### CODE REVIEW

File: custom/icds_reports/reports/awc_reports.py [hunk 1]
```diff
@@ -982,7 +982,7 @@ def get_awc_report_beneficiary(start, length, draw, order, filters, month, two_b
     filters['open_in_month'] = 1
     filters['valid_in_month'] = 1
     if filters.get('age_in_months__range') is None:
-        filters['age_in_months__lte'] = 60
+        filters['age_in_months__lte'] = 72
```
Comment (czue) [inline custom/icds_reports/reports/awc_reports.py]:
```
test_commcarehq_icds_ucr=# select count(*) from child_health_monthly where awc_id='a18' and month='2017-05-01' and open_in_month=1 and valid_in_month=1 and age_in_months <= 72; count 
-------
    32
(1 row)

test_commcarehq_icds_ucr=# select count(*) from child_health_monthly where awc_id='a18' and month='2017-05-01' and open_in_month=1 and valid_in_month=1 and age_in_months <= 60;
 count 
-------
    27
(1 row)
```

File: custom/icds_reports/tests/reports/test_awc_reports.py [hunk 2]
```diff
@@ -2474,13 +2474,13 @@ def test_awc_report_beneficiary_data_without_data(self):
...
                 "last_month": "May 2017",
-                "recordsTotal": 27,
+                "recordsTotal": 32,
                 "months": [
                     "May 2017",
                     "Apr 2017",
                     "Mar 2017"
                 ],
-                "recordsFiltered": 27,
+                "recordsFiltered": 32,
```
Comment (czue) [inline custom/icds_reports/tests/reports/test_awc_reports.py]:
```
test_commcarehq_icds_ucr=# select count(*) from child_health_monthly where awc_id='a18' and month='2017-05-01' and open_in_month=1 and valid_in_month=1 and age_in_months <= 72;
 count 
-------
    32
(1 row)

test_commcarehq_icds_ucr=# select count(*) from child_health_monthly where awc_id='a18' and month='2017-05-01' and open_in_month=1 and valid_in_month=1 and age_in_months <= 60;
 count 
-------
    27
(1 row)
```

**Comment by Rohit25negi** [comment]
Holding on until the questions on the tickets are answered.

**Comment by czue** [comment]
sounds like it's ok to proceed @Rohit25negi

**Review by Rohit25negi — APPROVED** [review]
Thanks @czue  for handling this.

**Pull request merged**